### PR TITLE
Add sass support for tilde

### PR DESCRIPTION
**Pull request by KaboomFox**
Fixes issue #322  False reports from scss files with `@import ~package-in-node_module/main.css`

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -929,19 +929,22 @@
       "version": "4.0.0",
       "resolved": "https://registry.npmjs.org/arr-diff/-/arr-diff-4.0.0.tgz",
       "integrity": "sha1-1kYQdP6/7HHn4VI1dhoyml3HxSA=",
-      "dev": true
+      "dev": true,
+      "optional": true
```
Comment (KaboomFox) [inline package-lock.json]: Changes in this file fixe `high severity vulnerability` found by npm audit.

**Comment by rumpl** [comment]
Hey @KaboomFox, thanks for making this PR, could you just address the linting problem that makes the [build fail](https://ci.appveyor.com/project/lijunle/depcheck/builds/22710695/job/fqs801ay6b15p2t0#L214)?

**Comment by KaboomFox** [comment]
@rumpl will fix! Thanks!

**Comment by rumpl** [comment]
Thanks!

**Comment by rumpl** [comment]
Version 0.7.2 with tilde is on npm now, thanks @KaboomFox

**Pull request merged**